src/cvl/custom_validation/sonic_sflow.go

```diff
@@ -0,0 +1,31 @@
+package custom_validation
+
+import (
+	"net"
+	util "cvl/internal/util"
+ )
+
+
+//Validate ip address of sflow collector ip 
+func (t *CustomValidation) ValidateIp(vc *CustValidationCtxt) CVLErrorInfo {
+	ip := net.ParseIP(vc.YNodeVal)
```

Comment:
Please add the following code here  as we don't want any validation for DELETE.

        if (vc.CurCfg.VOp == OP_DELETE) {
                 return CVLErrorInfo{ErrCode: CVL_SUCCESS}
        }

Reply:
Please also fully test out the delete case

Reply:
Done. Test result is shown below.
![sflow_del](https://user-images.githubusercontent.com/31632515/73404534-95981d80-42a6-11ea-8d6f-2ab52d065277.PNG)